File: Quivi.Pos.Api/Dtos/Transaction.cs
namespace Quivi.Pos.Api.Dtos
{
    public class Transaction
    {
        public required string Id { get; init; }
        public required string ChannelId { get; init; }
        public string? SessionId { get; init; }
        public string? EmployeeId { get; init; }
        public string? RefundEmployeeId { get; init; }
        public string? CustomChargeMethodId { get; init; }
        public decimal Payment { get; init; }
        public decimal Tip { get; init; }
        public decimal RefundedAmount { get; init; }
        public bool IsSynced { get; init; }
        public bool IsFreePayment { get; init; }
        public string? Email { get; init; }
        public string? VatNumber { get; init; }

        public DateTimeOffset CapturedDate { get; init; }
        public DateTimeOffset LastModified { get; init; }

    }
}

File: quivi.pos.react.app/src/hooks/api/Dtos/transactions/Transaction.ts
export interface Transaction {
    readonly id: string;
    readonly isSynced: boolean;
    readonly channelId: string;
    readonly employeeId?: string;
    readonly customChargeMethodId?: string;
    readonly payment: number;
    readonly tip: number;
    readonly refundedAmount: number;
    readonly isFreePayment: boolean;
    readonly sessionId?: string;
    readonly email?: string;
    readonly vatNumber?: string;
    readonly refundEmployeeId?: string;
    readonly capturedDate: string;
    readonly lastModified: string;
}